tests/all/func.rs

```diff
@@ -1966,7 +1966,7 @@ fn typed_v128(config: &mut Config) -> anyhow::Result<()> {
     Ok(())
 }
 
-#[wasmtime_test(wasm_features(simd))]
+#[wasmtime_test]
```

Comment:
Not sure if we want to keep the `wasm_features(simd)` check on here since this is a SIMD test. I removed it so the test would run on Winch.